tests/Extensions/Hosting.Services.Web.UnitTests/ActivityEnrichmentMiddlewareTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Omex.Extensions.Abstractions.Activities;
+using Microsoft.Omex.Extensions.Abstractions.Activities.Processing;
+using Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares;
+using Microsoft.Omex.Extensions.TimedScopes.UnitTests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Hosting.Services.Web.UnitTests
+{
+	[TestClass]
```

Comment:
Suggesting [TestCategory()]

Reply:
Not sure that it's needed, but added category, thank you